### DISCUSSION TITLE
Option to disable checking for valid segments

### DISCUSSION THREAD

**Discussion by Tydoki** [MacielG1/VodRecovery#3] [General]
Can we have an option to disable checking for valid segments? Often for long vods this process can hang and slow down extremely. I've had this process take upwards of 20 minutes in some cases.

**Comment by MacielG1** [comment]
> Can we have an option to disable checking for valid segments? Often for long vods this process can hang and slow down extremely. I've had this process take upwards of 20 minutes in some cases.

Currently, you can open the file settings.json inside the folder config and change the option CHECK_SEGMENTS to false. In the future, I could add an option in the menu to toggle that setting.

**Reply by Tydoki** [reply]
> Currently, you can open the file settings.json inside the folder config and change the option CHECK_SEGMENTS to false.

Yup that worked, Thank you!
